ontology/uco/core/core.ttl

```diff
@@ -378,6 +378,17 @@ core:UcoInherentCharacterizationThing
 	sh:targetClass core:UcoInherentCharacterizationThing ;
 	.
 
+core:UcoInherentCharacterizationThing-disjointWith-UcoObject-shape
+	a sh:NodeShape ;
+	sh:message "observable:UcoInherentCharacterizationThing and observable:UcoObject are disjoint classes.  Assigning both types to a single node will be an error in UCO 2.0.0."@en ;
```

Comment:
"... will be an error *as of* UCO 2.0.0"

Reply:
Fine by me.  I'll push a patch later today.

Reply:
Ah, nevermind, you've merged.  Thank you for merging.

Reply:
We both seem to have missed that "observable" was used in this message instead of "core".  Whoops.